api/src/main/java/org/openmrs/module/aijar/AijarActivator.java

```diff
@@ -146,7 +148,8 @@ public void started() {
 
             // update the name of the default health center with that stored in the global property
             Location healthCenter = locationService.getLocationByUuid("629d78e9-93e5-43b0-ad8a-48313fd99117");
-            healthCenter.setName(administrationService.getGlobalProperty(AijarConstants.GP_HEALTH_CENTER_NAME));
+            healthCenter.setName(administrationService.getGlobalProperty(GP_HEALTH_CENTER_NAME));
```

Comment:
@slubwama Can you add code to show a message if certain properties have not been changed and are still the defaults? These include:

1. Health Center Name 

2. DHIS2 Organization UUID 

3. National Health Provider Identifier